src/components/AsideHeader/AsideHeader.scss

```diff
@@ -170,6 +170,20 @@ $block: '.#{variables.$ns}aside-header';
         }
     }
 
+    &__logo-button-wrapper {
+        width: var(--gn-aside-header-min-width);
+    }
+
+    &__logo-button[class] {
+        &,
+        .g-button__icon {
```

Comment:
Is there an option not to use a class from uikit?

Reply:
As I see there is no api for that for now